server/src/core/utilities.ts

```
@@ -435,15 +443,32 @@ function formatNumber(n: number) {
   });
 }
 
+function generatePlayerListPayload(connectionIDs: string[]) {
+  const payload = [];
+  for (let connectionID of connectionIDs) {
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'connectionID' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)